setup.py

```diff
@@ -55,6 +55,7 @@
     "GPUtil>=1.4.0",
     "protobuf>=3.12.2,<=3.20.3",
     "click>=7.1.2,!=8.0.0",  # latest version < 8.0 + blocked version with reported bug
+    "sparsetensors @ git+ssh://git@github.com/neuralmagic/sparsetensors.git",
```

Comment:
you can't build/publish a wheel with a dependency with git deps due to privacy restrictions